guide.gemspec

```diff
@@ -14,8 +14,12 @@ Gem::Specification.new do |s|
 
   s.files = Dir["{app,config,db,lib}/**/*", "LICENSE", "Rakefile", "README.rdoc"]
 
-  s.add_dependency "railties", ">= 4"
-  s.add_dependency "activemodel", ">= 4"
+  s.required_ruby_version = ">= 2.6"
+
+  s.add_dependency "railties", ">= 5.2"
+  s.add_dependency "actionpack", ">= 5.2"
+  s.add_dependency "actionview", ">= 5.2"
```

Comment:
I've also added `actionpack` and `actionview` as explicit dependencies. Previously, these were transitive dependencies, via `sprockets-rails`. I notice that code in `guide` is using classes provided by these gems so best to specify them explicitly.